.azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+variables:
+  CONDASOURCE: "~/miniconda3/etc/profile.d/conda.sh"
+
+jobs:
+- job: 'Testing'
+  displayName: 'Unit Testing With Programs'
+  pool:
+    name: qcengine
+  timeoutInMinutes: 360
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 3
+
+  steps:
+    - script: |
+        echo '##vso[task.setvariable variable=ENVPREFIX]$(Agent.TempDirectory)/testnv$(python.version)'
+      displayName: "Worker Variable Assignment"
+
+      # Additional info about the build
+    - script: |
+        uname -a
+        free -m
+        df -h
+        ulimit -a
+      displayName: "Build Info"
+
+    # Setup python environment
+    - script: |
+        source $(CONDASOURCE)
+        # Create test environment for package
+        conda env create -p $ENVPREFIX python=$(python.version) -f devtools/conda-envs/psi.yaml
+        conda activate $ENVPREFIX
+        # Build and install package
+        python setup.py develop --no-deps
+      displayName: "Conda Env Creation"
+
+    # Display MongoDB, Conda Environment, Python version
+    - script: |
+        sleep 5
```

Comment:
No need for sleep anymore?

Reply:
Probably not, since this is now its own script.